web-server/routes/calendar.test.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+const loadDigest = require('../../digest')
+
+describe('GET /calendar route', function() {
+  it('Renders a page with iframe containing calendar', function(done){
+    webServer()
+      .get('/calendar/')
+      .end(function (err, res) {
```

Comment:
In this app there is an established patten of never using abreviations like `err` and `res`. If you see some I'd like to change them. Here please user `error` and `response`